Cargo.toml

```diff
@@ -11,7 +11,7 @@ url = "1.5.1"
 serde = "1.0"
 serde_json = "1.0"
 serde_derive = "1.0"
-reqwest = "0.8"
+reqwest = { version = "0.9.15", default-features = false }
```

Comment:
@dylemma any idea why 0.9.15 was picked? It's far from the latest version, and it's not even the latest 0.9.x release...